requirements.yaml

```diff
@@ -19,3 +19,6 @@ dependencies:
   - conda-forge:pretty_html_table=0.9.dev0
   - conda-forge:pandas_schema=0.3.5
   - bioconda::trackhub=0.1.2019.12.24
+#  - bioconda::pysradb=0.11.0
+  - pip:
+      - git+https://github.com/saketkc/pysradb.git
```

Comment:
norns and beautifulsoup4 (and perhaps others) are no longer required now right?

nitpick: newline

Reply:
Yes, norns and bs4 you are correct! For the actual release we are waiting for pysradb 0.11.1, since that has some bug fixes we need, but is not released. So that will be replaced by the bioconda thing